### ISSUE TITLE
CUB Concentrator always thinks a player is concentrating.

### ISSUE THREAD

**Issue by AtreidesGhola** [death-save/combat-utility-belt#212]
Whenever a player casts a concentration spell, there is an output in the chat that says:

CUB Concentrator

(Character) cast a spell requiring Concentration while concentrating on another spell.  Concentration n the original spell is lost.

This occurs in all circumstances when casting a concentration spell, even if there is no concentration set on the actor token.

CUB applies the concentration correctly to the token, but then gives the warning - see screen shots with a PC casting Faerie Fire.
Before Casting - not concentrating on anything
<img width="1900" alt="CUBCon1" src="https://user-images.githubusercontent.com/66924036/86518757-d2da7b80-be01-11ea-9f0c-b2977a1764b3.PNG">

After Casting - concentration applied, but two warnings in the chat; also set not to warn about double concentration in CUBPuter.

<img width="1898" alt="CUBCon2" src="https://user-images.githubusercontent.com/66924036/86518760-d7069900-be01-11ea-9a93-8a4c1ef9aa02.PNG">

**To Reproduce**
This occurs when the "Automatically set Concentration Status" is enabled in the CUBPuter.

**Expected behavior**
That this message only appear when the actor is concentration on a spell.

**Environment Info (please complete the following information):**
 - Foundry Version: 0.6.4
 - CUB Version [e.g. 22]
 - Browser: Chrome
- OS Windows 10 Pro

**Comment by Rabalaba** [comment]
Is the Faerie Fire spell attaching to more than one target?

**Comment by AtreidesGhola** [comment]
It happens regardless of the spell - yes Faerie targets multiple actors, but this occurs with every spell that triggers concentration being applied - and it does it when the actor casting isn't targeting anything.

**Comment by AtreidesGhola** [comment]
Also, I think that this issue was asked about previously - see the other Issue here:

[](https://github.com/death-save/combat-utility-belt/issues/178)

**Comment by eclarke12** [comment]
@AtreidesGhola by chance do you have multiple GMs in your world when this happens?

**Comment by Linith** [comment]
I have this issue as well, now it doesn't even apply the condition anymore.

**Comment by AtreidesGhola** [comment]
> @AtreidesGhola by chance do you have multiple GMs in your world when this happens?

No, there's only me signed in as a GM - the problem still persists - always get multiple notifications for concentration, regardless of if the actor is concentrating already.  It happens with all concentration spells I've tried.

Note:  Using the MQoL improvements mod, and have MESS installed - all MESS elements excepting the spell template animations are turned off.  This behavior occurs if MQoL and MESS are uninstalled as well.
